Tiltfile

```
@@ -106,6 +106,30 @@ cmd_button(
     icon_name="dangerous",
 )
 
+# Inspired by https://github.com/grafana/slo/blob/main/Tiltfile#L72
+pod_engine_pytest_script = '''
+set -eu
+# get engine k8s pod name from tilt resource name
+POD_NAME="$(tilt get kubernetesdiscovery "engine" -ojsonpath='{.status.pods[0].name}')"
+kubectl exec "$POD_NAME" -- pytest .
```

```suggestion
kubectl exec "$POD_NAME" -- pytest . --ds=settings.ci-test
```

I'm wondering if there's a way we could also provide optional arguments to `pytest`? Similar to how we currently have [`make test-dev`](https://github.com/grafana/oncall/blob/dev/Makefile#L180-L182) and you can run something like this:
```bash
$ make test-dev ARGS="apps/api/tests/test_auth.py"
```

---

Added, thank you for the idea